### PR TITLE
Fixes #28978 - Add description to plugin roles

### PR DESCRIPTION
**Pull request by stejskalleos**
PR is part of  [Add default descriptions to all roles added from plugins](https://projects.theforeman.org/issues/28936) task.

### CODE REVIEW

**Comment by stejskalleos** [comment]
@adamruzicka can you please give me description for the roles? Thanks

**Comment by stejskalleos** [comment]
[test Redmine]

File: lib/foreman_tasks/engine.rb [hunk 1]
```diff
@@ -69,8 +69,8 @@ class Engine < ::Rails::Engine
         logger :dynflow, :enabled => true
         logger :action, :enabled => true
 
-        role 'Tasks Manager', [:view_foreman_tasks, :edit_foreman_tasks]
-        role 'Tasks Reader', [:view_foreman_tasks]
+        role 'Tasks Manager', [:view_foreman_tasks, :edit_foreman_tasks], 'TODO'
+        role 'Tasks Reader', [:view_foreman_tasks], 'TODO'
```
Comment (adamruzicka) [inline lib/foreman_tasks/engine.rb]: Gives the user permissions to inspect tasks

File: lib/foreman_tasks/engine.rb [hunk 2]
```diff
@@ -69,8 +69,8 @@ class Engine < ::Rails::Engine
         logger :dynflow, :enabled => true
         logger :action, :enabled => true
 
-        role 'Tasks Manager', [:view_foreman_tasks, :edit_foreman_tasks]
-        role 'Tasks Reader', [:view_foreman_tasks]
+        role 'Tasks Manager', [:view_foreman_tasks, :edit_foreman_tasks], 'TODO'
```
Comment (adamruzicka) [inline lib/foreman_tasks/engine.rb]: Gives the user permissions to inspect, cancel, resume and unlock tasks

**Comment by stejskalleos** [comment]
@adamruzicka updated & rebased.

File: lib/foreman_tasks/engine.rb [hunk 3]
```diff
@@ -69,8 +69,10 @@ class Engine < ::Rails::Engine
         logger :dynflow, :enabled => true
         logger :action, :enabled => true
 
-        role 'Tasks Manager', [:view_foreman_tasks, :edit_foreman_tasks]
-        role 'Tasks Reader', [:view_foreman_tasks]
+        role 'Tasks Manager', [:view_foreman_tasks, :edit_foreman_tasks],
+             'Role granting permissions to inspect tasks'
+        role 'Tasks Reader', [:view_foreman_tasks],
+             'Role granting permissions to inspect, cancel, resume and unlock tasks'
```
Comment (adamruzicka) [inline lib/foreman_tasks/engine.rb]: I'm not sure if it was me or you, but the descriptions should be swapped.

**Comment by stejskalleos** [comment]
[test foreman-tasks]

**Comment by adamruzicka** [comment]
Thank you @stejskalleos & @pondrejk !

**Pull request merged**